openwisp_monitoring/check/settings.py

```diff
@@ -6,9 +6,16 @@
     (
         ('openwisp_monitoring.check.classes.Ping', 'Ping'),
         ('openwisp_monitoring.check.classes.ConfigApplied', 'Configuration Applied'),
+        (
+            'openwisp_monitoring.check.classes.SnmpDeviceMonitoring',
+            'SNMP Device Monitoring',
+        ),
     ),
 )
 AUTO_PING = getattr(settings, 'OPENWISP_MONITORING_AUTO_PING', True)
+AUTO_SNMP_DEVICEMONITORING = getattr(
+    settings, 'OPENWISP_MONITORING_AUTO_SNMP_DEVICEMONITORING', True
```

Comment:
this should be `False`. As I explained in our last call, the addition of the check will be manual for now.